CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
 
 ##### Enhancements
 
-* None.
+* Add support for annotations.  
```

Comment:
Can you be more specific, like `Add support for C-language annotations.`?

Reply:
Done.